SA/design_concept/abstract.typ

```diff
@@ -1,5 +1,22 @@
 = Abstract
-Content: Problem description, goal, proceedure, results, recommendations
-(Keywords?)
+Most visual programming tools used to introduce children & young adults to the world of programming are made for imperative programming.
+Existing tools for functional programming lack in visual aesthetics or hide key concepts of the functional programming paradigm.
+
+To address this gap, a visual, block-based tool for functional programming should be designed.
+The focus of the project lies on finding a visual design of such a tool and then prove its feasibility in a proof of concept application.
+
+Before creating a visual design, existing visual programming tools are examined.
+The design development consists of two iterations:
+The first concept drafts are based on researched tools and evaluated using a survey.
+Using the received feedback, a new concept is created and implemented in a PoC.
```

Comment:
```suggestion
The development of the design is then approached in two iterations: 
In the first step, concept drafts are based on researched tools and evaluated using a survey.
Then, a new concept is created and implemented in a PoC using the received feedback.
```

Reply:
Is 'PoC' ok to use here in the abstract?

It's probably the first use of the abbreviation in text, but the abstract also is a bit special, so I don't know.

Reply:
I've no idea. I replaced it.